app/src/main/java/org/standardnotes/notes/SettingsActivity.kt

```diff
@@ -35,6 +38,24 @@ class SettingsActivity : BaseActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        // TODO does this make sense here?
+        ExportUtil.clearExports(this)
```

Comment:
I think so? I could see a scenario where the activity could be destroyed while you're composing the email.

Possibly during the onResume of the previous activity? But that's stranger timing, and doesn't future proof any design changes, which might not be an issue.